### PR TITLE
RNA: Settings Fix Up

### LINKED ISSUE #29835: Jetpack Plugin Visual Refresh: Settings routes and cards
### Major stuff merged! Just needs some follow-up now https://github.com/Automattic/jetpack/pull/30204#issuecomment-1520841435 

The settings routes. `/wp-admin/admin.php?page=jetpack#/settings`

Each card within settings has many variations: Active, Inactive, Upgrade needed, Feature managed by another plugin (SEO), sub options, etc.— there’s more than 50+ card variations in these settings. Since many are unique, like SEO tools, we refreshed almost every single variation. 

See more images, details, and links to Figma: p1HpG7-lx8-p2#settings

Image

### PR DESCRIPTION
**Pull request by nateweller**
Addresses https://github.com/Automattic/jetpack/pull/30204#issuecomment-1520841435

## Proposed changes:

* Updates styles surrounding the setting tabs, based on feedback from https://github.com/Automattic/jetpack/pull/30204#issuecomment-1520841435

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

https://github.com/Automattic/jetpack/pull/30204#issuecomment-1520841435

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Go through each item from the linked comment and review the updated setting screens.

## Screenshots

Screen Shot 2023-04-27 at 14 55 21-fullpage
Screen Shot 2023-04-27 at 14 55 29-fullpage
Screen Shot 2023-04-27 at 14 55 51-fullpage
Screen Shot 2023-04-27 at 14 55 58-fullpage
Screen Shot 2023-04-27 at 14 56 09-fullpage
Screen Shot 2023-04-27 at 14 56 42-fullpage

### CODE REVIEW

**Review by jeffgolenski — DISMISSED** [review]
Ace work my friend. This checks off every issue I reported in #30204. ship it!

File: projects/plugins/jetpack/_inc/client/components/select-dropdown/style.scss [hunk 1]
```diff
@@ -241,3 +241,53 @@
...
+		.notouch & {
+			&:hover {
+				color: var( --jp-green );
+			}
+			&.is-selected:hover {
+				color: $white;
+			}
+		}
+	}
+}
```
Comment (nateweller) [inline projects/plugins/jetpack/_inc/client/components/select-dropdown/style.scss:293]:
```suggestion
}

```

**Review by sergeymitr — CHANGES_REQUESTED** [review]
Hi @nateweller.

Looks great, just a couple minor things I noticed:

- [ ] I think the "external" icon shouldn't get wrapped to the next line:
Screen Shot 2023-04-28 at 17 29 16

- [x] "Comments -> Color scheme" dropdown still has that blue background:
Screen Shot 2023-04-28 at 17 34 01

**Comment by nateweller** [comment]
Thanks @sergeymitr! I fixed the dropdown. The external link icon wrapping lines isn't a new issue, so I think we can address it in a different PR.

**Review by sergeymitr — DISMISSED** [review]
👍

**Comment by jeffgolenski** [comment]
@nateweller this isn't a blocker to ship, but I noticed the [privacy settings page](https://github.com/Automattic/jetpack/pull/30204#issuecomment-1523739174) needs the updated toggle. You're good to ship from a design perspective at any time. Whenever we can get that change in is fine!

**Review by sergeymitr — APPROVED** [review]
Looks great!

**Pull request merged**